kevm

```diff
@@ -51,26 +51,26 @@ run_kompile() {
     fi
 
     case "${backend}" in
-        haskell) kompile_opts+=( --md-selector 'k & ! nobytes' )
-                 ;;
-        java)    kompile_opts+=( --md-selector 'k & ! bytes' )
-                 ;;
-        llvm)    kompile_opts+=( --md-selector 'k & ! nobytes'                                                 )
-                 kompile_opts+=( -ccopt -L${libff_dir}/lib -ccopt -I${libff_dir}/include                       )
-                 kompile_opts+=( -ccopt ${plugin_include}/c/plugin_util.cpp                                    )
-                 kompile_opts+=( -ccopt ${plugin_include}/c/crypto.cpp                                         )
-                 kompile_opts+=( -ccopt ${plugin_include}/c/blake2.cpp                                         )
-                 kompile_opts+=( -ccopt -g -ccopt -std=c++14                                                   )
-                 kompile_opts+=( -ccopt -lff -ccopt -lcryptopp -ccopt -lsecp256k1 -ccopt -lssl -ccopt -lcrypto )
-                 if [[ "$(uname -s)" == 'Linux' ]]; then
-                     kompile_opts+=( -ccopt -lprocps )
-                 elif [[ "$(uname -s)" == 'Darwin' ]]; then
-                     openssl_root="$(brew --prefix openssl)"
-                     brew_root="$(brew --prefix)"
-                     kompile_opts+=( -ccopt -I${brew_root}/include -ccopt -L/${brew_root}/lib -ccopt -I${openssl_root}/include -ccopt -L${openssl_root}/lib )
-                     kompile_opts+=( -ccopt -I${libcryptopp_dir}/include -ccopt -L/${libcryptopp_dir}/lib )
-                 fi
-                 ;;
+        haskell)   kompile_opts+=( --md-selector 'k & ! nobytes & ! node' --backend haskell)        ;;
+        java)      kompile_opts+=( --md-selector 'k & ! bytes   & ! node' --backend java )          ;;
+        llvm)      kompile_opts+=( --md-selector 'k & ! nobytes & ! node' )                         ;;&
+        node)      kompile_opts+=( --md-selector 'k & ! nobytes & ! standalone' --no-llvm-kompile ) ;;&
```

Comment:
What are the `&` at the end of these lines for?

Reply:
From the man page for bash: `Using ;;& in place of ;; causes the shell to test the next pattern list in the statement, if any, and execute any associated list on a successful match.`

It's so `llvm` and `node` get matched again for the commands that they share in common with each other.